capture/capture.go

```diff
@@ -311,6 +304,12 @@ func (l *Listener) read(handler Handler) {
 					}
 					return // can't find the linktype size
 				}
+			} else {
+				linkType = int(hndl.(*socket).h.LinkType())
+				linkSize = 14
+				if hndl.(*socket).isCooked() {
```

Comment:
What is cooked mean? 
Can you add a comment?

Reply:
it checks if link layer header is removed off of the packet buffer.(`by default we do this`)